src/store/reducers/listSlice.ts

```diff
@@ -1,23 +1,29 @@
 import { createSlice } from '@reduxjs/toolkit';
 
-type CategoryFilter = number | 'all';
-
-export interface Filters{
-  searchQuery: '',
-    categoryFilter: CategoryFilter
-    farmFilter: number[],
-    rateFilter:number[],
-    priceRange: number[],
+export interface Filters {
+  searchQuery: '';
+  categoryFilter: number | 'all';
+  farmFilter: number[];
+  rateFilter: number[];
+  priceRange: number[];
+  sortBy:
```

Comment:
better to create separate interface or type ISortBy {...}, interface Filters {... sortBy: ISortBy}

Reply:
okay